docker-compose.yml

```diff
@@ -125,6 +125,7 @@ services:
     environment:
       GF_SECURITY_ADMIN_PASSWORD: admin
       GF_USERS_ALLOW_SIGN_UP: 'false'
+      GF_SERVER_ROOT_URL: "${GF_SERVER_ROOT_URL}"
```

Comment:
This allows us to link to dashboards or log views with filter set ups etc. For example the shutdown issue: http://208.52.185.184:3000/goto/mX1lvEAMz

Reply:
(we may want to set up some domain aliases to avoid having to use the raw IPs)

Reply:
Added aliases on Cloudflare.